.github/workflows/ci.yml

```diff
@@ -48,8 +48,6 @@ jobs:
       run: |
         if ${{ matrix.prefer-lowest == 'prefer-lowest' }}; then
           composer update --prefer-lowest --prefer-stable
-        elif ${{ matrix.php-version == '8.1' }}; then
```

Comment:
Looks like 8.2 isn't added to the matrix.

Reply:
whoops, missed that one 😅